wdl2cwl/tests/wdl_files/BuildCembaReferences.wdl

```diff
@@ -79,15 +81,15 @@ task BuildBisulfiteReferences {
       echo "No monitoring script given as input" > monitoring.log &
     fi
 
-    python /build_bisulfite_references.py \
+    python3 /build_bisulfite_references.py \
       --input-fasta ~{fasta_input} \
       --forward-convert-out ~{fwd_converted_reference_fasta_output_name} \
       --reverse-convert-out ~{rev_converted_reference_fasta_output_name}
   >>>
 
   runtime {
     docker: "quay.io/broadinstitute/bisulfite-references:1.0"
-    # disks: "local-disk " + ceil(3.5 * (if input_size < 1 then 1 else input_size)) + " HDD"
+    #disks: "local-disk " + ceil(3.5 * (if input_size < 1 then 1 else input_size)) + " HDD"
```

Comment:
If I uncomment this I get `BuildCembaReferences.wdl:92:33: 'NoneType' object has no attribute 'value'` Can you investigate @Th3nn3ss ?

Reply:
For this, are we still using `$(expr / 1024)`